src/dashboard/src/wsgi.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+
+application = get_wsgi_application()
```

Comment:
Yes, that's being ignored purposely. There might be more changes after testing, but this is our first barrier.